client/state/plans/assembler.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+export const createPlanObject = plan => {
+	return {
+		androidSku: String( plan.android_sku ),
+		appleSku: String( plan.apple_sku ),
+		available: String( plan.available ),
+		billPeriod: Number( plan.bill_period ),
+		billPeriodLabel: String( plan.bill_period_label ),
+		cost: Number( plan.cost ),
+		capability: String( plan.capability ),
```

Comment:
I've implemented assembler patter because I saw it before in other redux implementations and I like the idea to set the property type. However I agree with you specially because the gotten data from server-side sometimes is inconsistent.